src/binding/c/mpit_api.txt

```diff
@@ -0,0 +1,515 @@
+# vim: set ft=c:
+
+MPI_T_category_changed:
+    .desc: Get the timestamp indicating the last change to the categories
+/*
+    Notes:
+    If two subsequent calls to this routine return the same timestamp, it is guaranteed that
+    the category information has not changed between the two calls. If the timestamp retrieved
+    from the second call is higher, then some categories have been added or expanded.
+*/
+{
+    *stamp = cat_stamp;
+}
+
+MPI_T_category_get_categories:
+    .desc: Get sub-categories in a category
+{ -- error_check -- indices
+    if (len != 0) {
+        MPIR_ERRTEST_ARGNULL(indices, "indices", mpi_errno);
```

Comment:
It looks like `MPIR_ERR` macros are mistakenly used in this file.

Reply:
FWIW, I think the rest of this is OK. If the error macro stuff gets cleaned up, we can probably merge this.

Reply:
Fixed.